pkg/apis/gatewayapi/v1beta1/doc.go

```diff
@@ -1,4 +1,4 @@
-// Package v1alpha2 contains API Schema definitions for the
+// Package v1beta1 contains API Schema definitions for the
```

Comment:
it looks like its also missing:
https://github.com/fluxcd/flagger/blob/eee3607ab778dded8ee1cd2ba8d37f2b53d39fa8/pkg/apis/gatewayapi/v1alpha2/doc.go#L1C1-L2C1

could you add this as well?